debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ShrCCommand.java

```diff
@@ -1001,19 +1005,34 @@ private void dbgShrcPrintAllStats(PrintStream out, J9JavaVMPointer vm, J9SharedC
 					}
 					if (rwOffset.eq(0)) {
 						byteDataLen += len.longValue();
-						if ((statTypes & BYTE_STATS) != 0) {
+						if (((statTypes & BYTE_STATS) != 0) 
+							|| ((statTypes & STARTUPHINT_STATS) != 0)
+						) {
+							if (((statTypes & STARTUPHINT_STATS) != 0) 
```

Comment:
I think this should be written differently. It assumes that only one type of STATS can be enabled at the same time, which may not be true in the future. It can check if BYTE_STATS is disabled.